precompiles/xcm/XCM_v2.sol

```diff
@@ -0,0 +1,80 @@
+pragma solidity ^0.8.0;
+
+/**
+ * @title XCM interface.
+ */
+interface XCM {
+    // A multilocation is defined by its number of parents and the encoded junctions (interior)
+    struct Multilocation {
+        uint8 parents;
+        bytes[] interior;
+    }
+    
+    /**
+     * @dev Withdraw assets using PalletXCM call.
+     * @param asset_id - list of XC20 asset addresses
+     * @param asset_amount - list of transfer amounts (must match with asset addresses above)
+     * @param beneficiary - Multilocation of beneficiary in respect to destination parachain
+     * @param destination - Multilocation of destination chain
+     * @param fee_index - index of asset_id item that should be used as a XCM fee
+     * @return bool confirmation whether the XCM message sent.
+     *
+     * How method check that assets list is valid:
+     * - all assets resolved to multi-location (on runtime level)
+     * - all assets has corresponded amount (lenght of assets list matched to amount list)
+     */
+    function assets_withdraw(
```

Comment:
With the addition of XTokens, do we still need to have 2 separate calls, one for native asset and one for foreign asset?